src/styles/v2.css

```
@@ -0,0 +1,31 @@
+:root {
+  --dsdl-white: #fff;
+  --dsdl-black: #101010;
+}
+
+.footer {
+  background-color: var(--dsdl-black);
+  color: var(--dsdl-white);
+  padding: 2rem 1.5rem;
+}
+
+@media (min-width: 992px) {
+  .footer {
+    padding: 2rem 5rem;
+  }
+}
+
+.footer .underlined-link {
+  text-decoration: underline;
+}
+
+.footer a {
+  color: inherit;
+  text-decoration: none;
+}
+
+.footer hr {
+  margin-left: 1.5rem;
+  margin-right: 1.5rem;
```

This is too much when taking into account the `.row-gap-3` on the container already adding `1em`.


```suggestion
  margin-left: 0.5rem;
  margin-right: 0.5rem;
```

---

ah! i can fix that. 🤙

---

fixed via 10fb299